internal/cli/project_destroy.go

```diff
@@ -56,12 +58,19 @@ func (c *ProjectDestroyCommand) Run(args []string) int {
 			return 1
 		}
 	}
-	_, err = c.project.DestroyProject(c.Ctx, &pb.Job_DestroyProjectOp{
-		Project:              &pb.Ref_Project{Project: project.Project.Name},
-		SkipDestroyResources: c.skipDestroyResources,
-	})
+
+	if project.Project.DataSource == nil {
+		_, err = c.project.Client().DestroyProject(c.Ctx, &pb.DestroyProjectRequest{
+			Project: c.project.Ref(),
+		})
+	} else {
+		_, err = c.project.DestroyProject(c.Ctx, &pb.Job_DestroyProjectOp{
+			Project:              &pb.Ref_Project{Project: project.Project.Name},
+			SkipDestroyResources: c.skipDestroyResources,
+		})
```

Comment:
This confused me at first - I wasn't sure what the difference was between these two `DestroyProject` funcs. Maybe a quick doc comment note about how one is a client func that queues a DestroyProject operation, and the other is an API call to the server that deletes the projects record from the db with no job?